content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,34 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
```

Comment:
Can we move this to the end of the list for "Prisma Client"?
It's not sorted alphabetically.
<img width="163" alt="image" src="https://github.com/prisma/docs/assets/183673/72aeba88-b948-400a-83c5-9d6aa63ab9bf">